TUnit.Analyzers/TUnit.Analyzers.csproj

```diff
@@ -2,7 +2,7 @@
   <PropertyGroup>
     <TargetFrameworks>netstandard2.0</TargetFrameworks>
     <Nullable>enable</Nullable>
-    <LangVersion>latest</LangVersion>
+    <LangVersion>preview</LangVersion>
```

Comment:
@MarkCiliaVincenti here for example - Why does this need to be preview now?

Reply:
Fixed.